### ISSUE TITLE
Olux's Prized bow better than the rest

### ISSUE THREAD

**Issue by SpitfiresThunder** [Wynncraft/Issues#3573]
So, the weapon merchant in Olux has these Legendary weapons, all called "Olux's Prized ____". To get these, one much complete the quest "The Shadow of the Beast" and trade in one Crushed Bigfoot Bone. While scrolling through all the weapon stats, I noticed something...distasteful.

Let us run over the weapons' soft-stats for a second, shall we?

"Olux's Prized Dagger" --

+5 Strength
+35/3s Poison
+10% Water Defense
+1/4s Mana Regen
+5 Intelligence
-10% Fire Damage
+15% XP Bonus
+10% Thorns
+10% Earth Damage
-10% Air Damage.
![2016-01-06_16 16 16](https://cloud.githubusercontent.com/assets/16551534/12155168/c0426b00-b491-11e5-80f5-7c51c57c2a9a.png)

"Olux's Prized Spear" --

+5 Strength
+35/3s Poison
+10% Water Defense
+1/4s Mana Regen
+5 Intelligence
-10% Fire Damage
+15% XP Bonus
+10% Thorns
+10% Earth Damage
-10% Air Damage.
![2016-01-06_16 16 18](https://cloud.githubusercontent.com/assets/16551534/12155188/dd218094-b491-11e5-9615-eea27b4bed77.png)

"Olux's Prized Wand" -- 

+5 Strength
+35/3s Poison
+10% Water Defense
+1/4s Mana Regen
+5 Intelligence
-10% Fire Damage
+15% XP Bonus
+10% Thorns
+10% Earth Damage
-10% Air Damage.
![2016-01-06_16 16 19](https://cloud.githubusercontent.com/assets/16551534/12155200/ea8a0f6c-b491-11e5-92f8-c6032f4c5fb1.png)

Based on the weapons so far, you'd probably expect the Bow to have the same stats, right? Well, let us find out:

"Olux's Prized Bow" --

+5 Strength
+35/3s Poison
+10% Water Defense
+1/4s Mana Regen
+5 Intelligence
-10% Fire Damage
+15% XP Bonus
+10% Thorns
+10% Earth Damage
-10% Air Damage.

And...hold on a second, what's this? There's something else? What could it possibly...
...
+15% Loot Bonus
![2016-01-06_16 16 20](https://cloud.githubusercontent.com/assets/16551534/12155228/23a82b08-b492-11e5-8c71-12afc10ee94f.png)

Seriously? If there wasn't a set of weapons that showed more blatant Archer/Hunter bias, I don't know what is.

In other words, Archer/Hunter players get a "special bonus" for completing the quest, simply because they're playing Archer/Hunter. Why? They already have the highest damaging weapons (which, sort of makes sense), the most range (which makes complete sense), the highest Damage spell (Creeper Arrow/Arrow Bomb, which makes no sense), and the highest Damage Per Second spell (Arrow Storm, which makes no sense either.) Literally, two or even one shot of an Arrow Bomb, and Bigfoot dies INSTANTLY.

Basically, they have the easiest time completing the quest, and so they get extra bonuses on their prize weapon for it. Of course, I may just be assuming here and blowing nothing but hot air...but for some reason, I doubt this to be the case.

TL;DR -- The "Olux's Prized Bow" gets an extra stat of +15% Loot Bonus that it doesn't deserve. Add it to the rest of the weapons, or remove it from the bow.

**Comment by Hankyana** [comment]
Don't look down on archers like that ;w; 
But yeah, its probably a mistake and will be fixed in accordance to what you stated.

**Comment by DarkDemonDragon** [comment]
Hmm perhaps you just got lucky? Not all IDs are guaranteed. Sometimes items that usually have a negative ID just don't because of pure luck. Someone should double check to see if this is a 100% thing with the bow.

**Comment by SpitfiresThunder** [comment]
Demon, these items are per-id'd items, sold at the item buyer in Olux.

In other words, those stats are 100% guaranteed every time you open his merchant inventory.

**Comment by DarkDemonDragon** [comment]
Yea just confirmed this.

**Comment by lXetnosl** [comment]
I recognized that as well... but it didnt really bother me cuz im archer xD but yes its pretty unfair i have to admit that... and nope that wasnt luck.

**Comment by Trixter313** [comment]
Is this still the case?

**Comment by SpitfiresThunder** [comment]
Yup, still is.

**Comment by Dar-krusos** [comment]
Yeah I found this out and tested it a month or so ago. Currently, the bow is better than the lv.70 legendary bow, Maelstrom. Jungle Artifact, however, has better stats than both the Olux's Prized Bow and Maelstrom, however I still prefer Olux's Prized Bow for the high XP bonus.

**Comment by DarkDemonDragon** [comment]
@Dar-krusos better in terms of general IDs or just the XP bonus? The unique bow Hope can ID to at least 25% XP I think (I have a 22% XP Hope), so that's another option to use.

**Comment by Dar-krusos** [comment]
No, not in aspect of ID's. In terms of damage, raw and also including with spell damage, it fares better. In terms of max ID's, it doesn't have the highest, but as an all-rounder, it is far superior than every bow besides Bob's Mythic (and Mythic bows).

**Comment by DarkDemonDragon** [comment]
@Dar-krusos wow seriously? That's insane ._.

**Comment by Odnod** [comment]
This has been fixed. They now all have +15% xp bonus. 

If someone else could get screenshots to truly confirm this that would be great.

**Comment by ThunderRemix** [comment]
@Trixter313 I think this "issue" can be closed now.

**Comment by Trixter313** [comment]
@ThunderRemix I just checked the merchant and it still has the bow that has +15% loot bonus.

**Comment by Moe-Ronickah** [comment]
I thought that this was fixed a long time ago . . . I just pmed Salted about it.

**Comment by ThunderRemix** [comment]
@Moe-Ronickah @Trixter313 Was this fixed yet? o_o

**Comment by Moe-Ronickah** [comment]
I just went to the Olux Weapon Merchant, and the kindly merchant proudly shewed me his collection of Olux's Prized Weapons.  Lo and behold, the olde Bow no longer has a Loot bonus.  This issue be fixed.
